app/3.0/docs/glossary.md

```diff
@@ -78,24 +53,34 @@ The HTML actually loaded by the browser to start the application. Usually `index
 application built with an [app shell](#app-shell) architecture, the entrypoint is loaded for many 
 different URLs.
 
+### ES (ES5, ES6/ES2015)
+
+ECMAScript. A specification created to standardize JavaScript. 
+
+ES5 and ES6 (ES2015) are editions of ECMAScript; ES6 adds features to ES5, such as [imports](#import).
+
+The Web Components standards and the Polymer library are based on ES6 JavaScript features. Polymer CLI can compile JavaScript code that uses newer features to ES5 so that older browsers can still run your app.
+
+### ES modules
```

Comment:
AKA "Standard JavaScript Modules"

Reply:
Elsewhere in the docs we've used ES6 modules, rather than just ES modules, I think.

Reply:
Nm. I think ES modules is good. Will work on standardizing that.

Reply:
Maybe add (to Justin's point) that ES modules are a JavaScript language feature added in ES6, so they're available natively in newer browsers unlike AMD modules.